### PR TITLE
fix: trimEnd polyfill (WEBAPP-5509)

### PR DESCRIPTION
**Pull request by ffflorian**

### CODE REVIEW

File: app/script/util/util.js [hunk 1]
```diff
@@ -148,6 +148,19 @@ z.util.getFileExtension = filename => {
...
+ * Trim end of a string.
+ * @param {string} text - text to trim
+ * @returns {string} Trimmed string
+ */
+z.util.trimEnd = text => {
+  if (_.isString(text)) {
+    return text.replace(/\s*$/, '');
+  }
+
+  return '';
```
Comment (gregor) [inline app/script/util/util.js]: Can easily be place in a ternary

File: app/script/util/util.js [hunk 2]
```diff
@@ -148,6 +148,19 @@ z.util.getFileExtension = filename => {
   return filename.substr(filename.lastIndexOf('.') + 1);
 };
 
+/**
+ * Trim end of a string.
+ * @param {string} text - text to trim
+ * @returns {string} Trimmed string
+ */
+z.util.trimEnd = text => {
+  if (_.isString(text)) {
```
Comment (atomrc) [inline app/script/util/util.js]: I would not put safeguards here. We probably want to know if we call this with an unexpected input

Comment (atomrc) [reply app/script/util/util.js]: Also how about do feature detection here.

```
return text.trimEnd ? text.trimEnd() : text.replace(...)
```

**Comment by ffflorian** [comment]
@gregor @atomrc What about the safeguard? Yes or no?

**Comment by bennycode** [comment]
Going forward we should investigate in using [Babel](https://babeljs.io/) for the webapp code.

File: app/script/util/util.js [hunk 3]
```diff
@@ -148,6 +148,15 @@ z.util.getFileExtension = filename => {
...
 };
 
+/**
+ * Trim end of a string.
+ * @param {string} text - text to trim
+ * @returns {string} Trimmed string
+ */
+z.util.trimEnd = text => {
+  return _.isString(text) ? text.replace(/\s*$/, '') : '';
+};
```
Comment (gregor) [inline app/script/util/util.js]: Arrow function fits into single line.

**Comment by ffflorian** [comment]
Should this go to `StringUtil`?

**Comment by atomrc** [comment]
Yes

**Pull request merged**